pkgs/development/python-modules/tensorflow/default.nix

```
@@ -267,7 +267,7 @@ let
 
       # cudaSupport causes fetch of ncclArchive, resulting in different hashes
       sha256 = if cudaSupport then
-        "1rbzvir569ch33bpvm47byx954vr80rjnzrbgs4rg3bqww73hr3k"
+        "1dpfyagcx1p4vzfabsl9v5i75332vv2mhk90897i75k6ciq292qs"
```

I got a different hash :(
```
hash mismatch in fixed-output derivation '/nix/store/khcwjcxv7l1w2n5w4rxipj0qrpqqy0ma-tensorflow-gpu-1.14.0-deps':
  wanted: sha256:1dpfyagcx1p4vzfabsl9v5i75332vv2mhk90897i75k6ciq292qs
  got:    sha256:1jrf5m1daazwawgvlv0z21qyf1l5aa4l19bfh961pll325xrh76h
cannot build derivation '/nix/store/rc1jhdpgd0wckwqmh5javig2cca419yr-tensorflow-gpu-1.14.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/djpai7nalflmdmx5cry9z1wizb30ivbg-python3.7-tensorflow-gpu-1.14.0.drv': 1 dependencies couldn't be built
```
```suggestion
        "1jrf5m1daazwawgvlv0z21qyf1l5aa4l19bfh961pll325xrh76h"
```

---

however, i confirmed that `python3Packages.tensorflowWithCuda` doesn't build on master
```
Loading: 446 packages loaded
installing
error: getting attributes of path '/nix/store/khcwjcxv7l1w2n5w4rxipj0qrpqqy0ma-tensorflow-gpu-1.14.0-deps/aws/aws-cpp-sdk-acm/include/aws/acm/model/Tag.h': No such file or directory
```

---

Ok, now I also get the same hash as you do, maybe it's updated a lot or impure. Also,
```-r--r--r-- 1 root root 3413 Jan  1  1970 /nix/store/khcwjcxv7l1w2n5w4rxipj0qrpqqy0ma-tensorflow-gpu-1.14.0-deps/aws/aws-cpp-sdk-acm/include/aws/acm/model/Tag.h```
so maybe you ran out of disk-space or something? Either way it seems this is not a stable hash. We could commit it now, or close, but it'll probably break again I guess.
People who need it can still go with tensorflow-bin so it's probably not urgent.